scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -234,7 +243,26 @@ def make_extract_table(fq_pet_vet_dataset,
               ):
   try:
     global client
-    default_config = QueryJobConfig(labels={ "id" : f"test_cohort_export_{output_table_prefix}"}, priority="INTERACTIVE", use_query_cache=False )
+    # this is where a set of labels are being created for the cohort extract. Do we want the hardcoded one to be different?
+    query_labels_map = {}
+    query_labels_map["id"]= f"test_cohort_export_{output_table_prefix}"
+    # query_labels is string that looks like 'label1=val1, label2=val2'
+    if query_labels != None:
+        query_labels_list = query_labels.split(",")
+        for query_label in query_labels_list:
+          kv = query_label.split("=", 2)
+          key = kv[0].strip().lower()
+          value = kv[1].strip().lower()
+          query_labels_map[key] = value
+
+
+      #import re
+      #if !bool(re.match(r"[a-z0-9_-]+$", key)): !True -- throw an error
+      #    raise ValueError(f"label key did not pass validation")
+
+    #Default QueryJobConfig will be merged into job configs passed into the query method.
+    # TODO I'm worried about how well the labels will be merged....
+    default_config = QueryJobConfig(labels=query_labels_map, priority="INTERACTIVE", use_query_cache=False )
```

Comment:
does this get overwritten when you define labels later? how did you resolve this?

Reply:
it doesn't get overwritten (see screenshot with default and query based labels)
in order to make sure they dont get overwritten, any new labels must be added to the `client._default_query_job_config.labels` that already exist

Reply:
The screenshots seem to be from the Java based tools, do you know how this python works?

Reply:
one of the screen shots is from the python I promise!!!!

Reply:
the java one is the second screen shot and is pretty boring because it's only one query
the python one is the first one and is much more interesting because it's a series of queries, and you can see the individual ones are now labeled to keep the initial label that was previously there and now can have the custom one (where I shove in my name) AND one that's based on the query specifically eg: "populate-final-extract-table"